fog/sample-paykit/src/client.rs

```diff
@@ -173,6 +176,18 @@ impl Client {
         self.tx_data.get_latest_block_version()
     }
 
+    /// Get the latest fee map
+    pub fn get_latest_fee_map(&mut self) -> Result<FeeMap> {
```

Comment:
Looks like this won't necessarily get the latest fee map since once it caches it, it no longer tries again. So that name is misleading.